packages/wallets/src/components/ExternalTradingPlatforms/AvailableCTraderAccountsList/AvailableCTraderAccountsList.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useCreateOtherCFDAccount } from '@deriv/api';
+import CTrader from '../../../public/images/ctrader.svg';
+import { SecondaryActionButton } from '../../SecondaryActionButton';
+import { TradingAccountCard } from '../../TradingAccountCard';
+import './AvailableCTraderAccountsList.scss';
+
+const AvailableCTraderAccountsList: React.FC = () => {
+    const { mutate } = useCreateOtherCFDAccount();
+
+    const createcTraderAccount = () => {
+        mutate({
+            payload: {
+                account_type: 'demo',
+                market_type: 'synthetic',
```

Comment:
market_type for cTrader, DerivX is all
```suggestion
                market_type: 'all',
```